Zip_Dockerfile.twig

```diff
@@ -0,0 +1,65 @@
+{% set shopwareVersion = imageName|split('_')[1] %}
+{% set shopwareUrl = imageName|split('_').slice(2, 5).join('_') %}
+{% set mysqlVersion = imageName|split('_')[5] %}
+{% set phpVersion = imageName|split('_')[6] %}
+{% set consentManagerVersion = imageName|split('_')[7] %}
+{% set consentManagerRequired = consentManagerVersion != 'none' %}
+
+# Check out shopware source
+FROM alpine/git:latest AS clone
+
+RUN wget --header="User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_0) AppleWebKit/600.1.17 (KHTML, like Gecko)a Version/8.0 Safari/600.1.17" https://releases.shopware.com/{{ shopwareUrl }}.zip -O install.zip && unzip install.zip -d /shopware
```

Comment:
Is a useragent strictly required?

Reply:
yes, if now the Route does not provide the package